public/images/home/subscription-background.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1680 305"><defs><linearGradient id="a" x1=".487" y1="6.843" x2=".513" y2="5.832"><stop offset="0" stop-color="#69c384"/><stop offset="1" stop-color="#49b17f"/></linearGradient><clipPath id="b"><path transform="translate(0 .001)" fill="none" d="M0 0h1680v305H0z"/></clipPath></defs><path d="M0 304.881V0h1679.589v304.881z" fill="url(#a)"/><g transform="translate(0 -.001)" clip-path="url(#b)"><ellipse cx="1272.833" cy="446.093" rx="537.047" ry="539.924" fill="#208e74" opacity=".4"/><ellipse cx="366.019" cy="-270.792" rx="497.325" ry="499.989" fill="#38a37a" opacity=".996"/><ellipse cx="814.168" cy="-48.753" rx="276.468" ry="277.949" fill="#15866b" opacity=".3"/></g></svg>
```

Comment:
Please note, the home folder is for the new homepage while tugHome is for the tug.tidb.io